components/Tabs/tests/unit/TabBarView/MDCTabBarViewItemViewTests.m

```diff
@@ -217,4 +217,70 @@ - (void)testSizeThatFitsForShortTitleExpectedImageLargerDimensionsLimitsHeight {
   XCTAssertEqualWithAccuracy(fitSize.height, kMinHeightOfTitleAndImageView, 0.001);
 }
 
+#pragma mark - -contentFrame
+
+- (void)testContentFrameForTextOnlyViewReturnsTitleLabelFittingFrame {
+  // Given
+  MDCTabBarViewItemView *itemView = [[MDCTabBarViewItemView alloc] init];
+
+  // When
+  itemView.titleLabel.text = @"Favorites";
+  itemView.bounds = CGRectMake(0, 0, 90, 48);
+
+  // Then
+  // Grab the `contentFrame` before layout to be sure it's calculated correctly at any time.
+  CGRect contentFrame = itemView.contentFrame;
+  [itemView layoutIfNeeded];
+  XCTAssertFalse(
+      CGRectEqualToRect(contentFrame, CGRectZero),
+      @"The `contentFrame` of an item view must not return CGRectZero when a title is set.");
```

Comment:
_Tip:_ @"(%@) is equal to (%@)", NSStringFromCGRect(contentFrame), NSStringFromCGRect(CGRectZero)

Reply:
Done.